### ISSUE TITLE
HBrowse: Images locked behind cookies

### ISSUE THREAD

**Issue by Palloxin** [tachiyomiorg/tachiyomi-extensions#8143]
### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.11.1](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### Tachiyomi version

0.11.1

### Android version

8

### Source information

Hbrowse 1.2.4

### Steps to reproduce

Open a manga e try to read it

### Expected behavior

It should load the pages of the manga in question

### Actual behavior

every page of the manga is like this:
![Immagine](https://user-images.githubusercontent.com/75091899/126035052-333f8eae-37db-492d-8afa-925a9d3ebf43.png)

### Other details

for other details someone already reported it in January but in the "feature request" and not as a "bug" and it is still open so: https://github.com/tachiyomiorg/tachiyomi-extensions/issues/5606

to see the images you have to open it the browser and click on the image attached in "Actual behavior".

**Comment by Soitora** [comment]
Not sure I wanna close due to duplicate as you've mentioned with #5606. It's up to @CarlosEsco how he want to handle this as the other one is locked for me to edit as well.

**Comment by CarlosEsco** [comment]
Is it not a feature request?  I assumed not every manga was blocked by cookie

**Comment by Soitora** [comment]
That's what I'd like to know too, I was assuming every manga was locked.

**Comment by Palloxin** [comment]
oh you are right, let's say half of them are locked

**Comment by CarlosEsco** [comment]
Ok so add this info to the other ticket

**Comment by Palloxin** [comment]
> 
> 
> Ok so add this info to the other ticket

but it is not my ticket

**Comment by CarlosEsco** [comment]
You can comment and add more info

**Comment by Palloxin** [comment]
"This conversation has been locked and limited to collaborators." it says @CarlosEsco

**Comment by CarlosEsco** [comment]
Try now

**Comment by Soitora** [comment]
Duplicate of #5606
